### DISCUSSION TITLE
AutoMapper 11 - unknown types

### DISCUSSION THREAD

**Discussion by feijoc** [LuckyPennySoftware/AutoMapper#3879] [NA]
What would be the equivalent of this in AutoMapper 11? I have lots of this code in a project that needs to be upgraded to v11 but I can't figure out if this is still possible? All the documentation suggests it isn't so can anyone confirm? 

var obj1 = GetSomeObject();
var obj2 = GetSomeOtherObject();

Mapper.Initialize(c => c.CreateMissingTypeMaps = true);
Mapper.Map(obj1, obj2, obj1.GetType(), obj2.GetType());

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/9.0-Upgrade-Guide.html#automapper-no-longer-creates-maps-automatically-createmissingtypemaps-and-conventions

**Reply by feijoc** [reply]
Yes that is the documentation I have seen so I assume you mean it is now not possible with v11. 

I guess I will downgrade AutoMapper back down to a version where this is achievable then.
